packages/react-components/react-progress/src/components/Progress/useProgressStyles.ts

```diff
@@ -114,75 +102,38 @@ const useBarStyles = makeStyles({
   },
 });
 
-const useTrackStyles = makeStyles({
-  base: {
-    gridRowStart: '2',
-    gridColumnStart: '1',
-    backgroundColor: tokens.colorNeutralBackground6,
-
-    '@media screen and (forced-colors: active)': {
-      ...shorthands.borderBottom('1px', 'solid', 'CanvasText'),
-    },
-  },
-  medium: {
-    height: barThicknessValues.medium,
-  },
-  large: {
-    height: barThicknessValues.large,
-  },
-});
-
 /**
  * Apply styling to the Progress slots based on the state
  */
 export const useProgressStyles_unstable = (state: ProgressState): ProgressState => {
-  const { indeterminate, thickness, percentComplete } = state;
+  const { max = 1.0, thickness, value } = state;
   const rootStyles = useRootStyles();
   const barStyles = useBarStyles();
-  const trackStyles = useTrackStyles();
-  const labelStyles = useLabelStyles();
-  const descriptionStyles = useDescriptionStyles();
   const { dir } = useFluent();
 
-  state.root.className = mergeClasses(progressClassNames.root, rootStyles.root, state.root.className);
+  state.root.className = mergeClasses(
+    progressClassNames.root,
+    rootStyles.root,
+    rootStyles[thickness],
+    state.root.className,
+  );
 
   if (state.bar) {
     state.bar.className = mergeClasses(
       progressClassNames.bar,
       barStyles.base,
-      indeterminate && barStyles.indeterminate,
-      indeterminate && dir === 'rtl' && barStyles.rtl,
+      value === undefined && barStyles.indeterminate,
+      value === undefined && dir === 'rtl' && barStyles.rtl,
       barStyles[thickness],
-      !indeterminate && barStyles.determinate,
-      !indeterminate && percentComplete > ZERO_THRESHOLD && barStyles.nonZeroDeterminate,
+      !!value && barStyles.determinate,
+      !!value && value > ZERO_THRESHOLD && barStyles.nonZeroDeterminate,
```

Comment:
This should be checking `value !== undefined` instead of `!!value`, so the determinate styling gets applied when value === 0.
```suggestion
      value !== undefined && barStyles.determinate,
      value !== undefined && value > ZERO_THRESHOLD && barStyles.nonZeroDeterminate,
```